### PR TITLE
CDAP-1300: ScheduleClient for listing, suspending, and resuming schedules

### PR DESCRIPTION
**Pull request by jwang47**
https://issues.cask.co/browse/CDAP-1300
http://builds.cask.co/browse/CDAP-DUT1037

### CODE REVIEW

File: cdap-client/src/main/java/co/cask/cdap/client/ScheduleClient.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
```
Comment (sagarkapare) [inline cdap-client/src/main/java/co/cask/cdap/client/ScheduleClient.java]: copyright - 2015

File: cdap-client-tests/src/test/java/co/cask/cdap/client/ScheduleClientTestRun.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
```
Comment (sagarkapare) [inline cdap-client-tests/src/test/java/co/cask/cdap/client/ScheduleClientTestRun.java]: Copyright - 2015

**Comment by sagarkapare** [comment]
Minor comments, otherwise LGTM :+1:

**Comment by albertshau** [comment]
lgtm

**Pull request merged**